common/djangoapps/student/models.py

```diff
@@ -1695,9 +1724,4 @@ class Meta:
         unique_together = (('code', 'user_profile'),)
 
     user_profile = models.ForeignKey(UserProfile, db_index=True, related_name='language_proficiencies')
-    code = models.CharField(
-        max_length=16,
-        blank=False,
-        choices=settings.ALL_LANGUAGES,
-        help_text=ugettext_lazy("The ISO 639-1 language code for this language.")
-    )
+    code = LanguageField(blank=False)
```

Comment:
Talk to devops to see if this will impact existing migration files for language proficiency.